packages/scm/src/browser/scm-tree-widget.tsx

```diff
@@ -630,13 +654,15 @@ export class ScmResourceComponent extends ScmElement<ScmResourceComponent.Props>
         }
     };
 }
+
 export namespace ScmResourceComponent {
     export interface Props extends ScmElement.Props {
         treeNode: ScmFileChangeNode;
         parentPath: URI;
         sourceUri: string;
         decoration: Decoration | undefined;
         colors: ColorRegistry;
+        isLight: boolean
```

Comment:
minor: use a more representative name:

```suggestion
        isLightTheme: boolean
```

Reply:
Yeah, makes more sense.